DataGenerator/DataGenerator.py

```diff
@@ -101,33 +111,31 @@ def __getitem__(self, i):
             if channel == 'Records':
                 records         = self.SubjectList.loc[:,self.clinical_cols].to_numpy()
                 data['Records'] = torch.as_tensor(records, dtype=torch.float32)
-
         if self.inference: return data
 
         else:
             label = self.SubjectList.loc[i,"xnat_subjectdata_field_map_"+self.config['DATA']['target']]
-            if self.config['DATA']['threshold'] is not None:  label = np.array(label > self.config['DATA']['threshold'])
+            if self.config['MODEL']['Classification_threshold'] is not None:  label = np.array(label > self.config['MODEL']['Classification_threshold'])
```

Comment:
We need to uniformize configuration tag, I think we should use the one we have right now.